constants/fokabotCommands.py

```diff
@@ -165,24 +175,40 @@ async def moderated(fro: str, channel_name: str, message: list[str]) -> str:
         # Turn on/off moderated mode
         # NOTE: this will raise exceptions.channelUnknownException if the channel doesn't exist
         await channelList.updateChannel(channel_name, moderated=enable)
-
-        return f'This channel is {"now" if enable else "no longer"} in moderated mode!'
+        userID = userUtils.getID(fro)
+        await rap_logs.send_rap_log(
+            userID,
+            f"has toggled moderated mode in {channel_name}.",
+        )
+        await rap_logs.send_rap_log_as_discord_webhook(
+            message=f"[{fro}](https://akatsuki.gg/u/{userID}) ({userID}) has toggled moderated mode in {channel_name}.",
+            discord_channel="ac_general",
+        )
+        response = (
+            f'This channel is {"now" if enable else "no longer"} in moderated mode!'
+        )
     except exceptions.channelUnknownException:
-        return "Channel doesn't exist. (??? contact cmyui(#0425))"
+        response = "Channel doesn't exist. (??? contact cmyui/tsunyoku)"
     except exceptions.moderatedPMException:
-        return "You are trying to put a private chat in moderated mode.. Let that sink in for a second.."
+        response = "You are trying to put a private chat in moderated mode.. Let that sink in for a second.."
+
+    return response
 
 
 @command(
     trigger="!kick",
     privs=privileges.ADMIN_KICK_USERS,
-    syntax="<target_name>",
+    syntax="<target_name> <reason>",
     hidden=True,
 )
 async def kick(fro: str, chan: str, message: list[str]) -> str:
     """Kick a specified player from the server."""
-    target = message[0].lower()
-    if target == glob.BOT_NAME.lower():
+    message = [m.lower() for m in message]
+    target = message[0]
+    reason = " ".join(message[1:])
+    userID = userUtils.getID(fro)
```

Comment:
```suggestion
    userID = await userUtils.getID(fro)
```

Reply:
if you're using vs code, i think enabling `type_checking_mode: basic` in your settings (ctrl+,) will point this error out

![image](https://i.cmyui.xyz/hlI5CTPRRlA.png)

Reply:
le baby intern didnt know this existed :sob: (fixed in b346c9ca6da9aa4aa864d14e140ae451cf67bbba)